eng/Build.props

```diff
@@ -1,9 +1,24 @@
-<Project>
-  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true' and '$(DotNetBuild)' != 'true'">
+<Project TreatAsLocalProperty="TargetRids">
+  <PropertyGroup>
+    <_AspireOnlyBuild Condition="'$(DotNetBuildFromSource)' != 'true' and '$(DotNetBuild)' != 'true'">true</_AspireOnlyBuild>
+
+    <BuildNative>true</BuildNative>
+
+    <BuildRid>$([System.Runtime.InteropServices.RuntimeInformation]::RuntimeIdentifier)</BuildRid>
+
+    <!-- this property is meant only for CI use. Use a colon separate list of RIDs here, like osx-x64:osx-arm64 .
```

Comment:
This says it is only meant for CI use, but you have it documented in the contributing.md. That doesn't seem right.